docs/layouts/partials/meta/google-analytics-async.html

```diff
@@ -94,3 +68,6 @@
     });
   };
 </script>
+<!-- END: Google Analytics -->
+{{- end -}}
+{{- end -}}
```

Comment:
This file is _copied_ from the theme and minimally modified to require users consent to the use of cookies before Google Analytics is activated. Hugo favors this file over the original in `themes/...`

This is consistent with the behavior prior to this PR. The only difference is that the hack is now part of our site code rather than a modification to the theme.